kotlin/openID4VP/src/commonMain/kotlin/io/mosip/openID4VP/common/Utils.kt

```
@@ -72,14 +94,19 @@ fun validate(
 
 inline fun <reified T> encodeToJsonString(data: T, fieldName: String, className: String): String {
     try {
-        val objectMapper = jacksonObjectMapper().setSerializationInclusion(JsonInclude.Include.NON_NULL)
+        val objectMapper =
+            jacksonObjectMapper().setSerializationInclusion(JsonInclude.Include.NON_NULL)
         return objectMapper.writeValueAsString(data)
     } catch (exception: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)